prov/efa/src/efa_fork_support.c

```diff
@@ -176,6 +176,29 @@ void efa_atfork_callback()
 	abort();
 }
 
+/**
+ * @brief flush all MR caches
+ *
+ * Going through all domains, and flush the MR caches in them
+ * until all inactive MRs are de-registered.
+ * This makes all memory regions that are not actively used
+ * in data transfer visible to the child process.
+ */
+void efa_atfork_callback_flush_mr_cache()
+{
+	struct dlist_entry *tmp;
+	struct efa_domain *efa_domain;
+	bool flush_lru = true;
```

Comment:
nit: I would combine this line with L197 as `while(ofi_mr_cache_flush(efa_domain->cache, true)`

Reply:
I wanted to use variable name to remind reader what the `true` means.  

Maybe I will change to 
```
ofi_mr_cache_flush(efa_domain->cache, true /* flush_lru */)
```